src/ResultSet.php

```diff
@@ -48,4 +53,21 @@ public function getIterator(): ArrayIterator
     {
         return new ArrayIterator($this->results);
     }
+
+    public function getErrors(): array
+    {
+        $errors = [];
+        foreach ($this->results as $attribute => $result) {
+            if (!$result->isValid()) {
+                $errors[$attribute] = $result;
```

Comment:
Did you mean to collect errors?

```suggestion
                $errors[$attribute] = $result->getErrors();
```

Reply:
done